plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotConfig.java

```diff
@@ -64,9 +64,9 @@
     private boolean aggregationPushdownEnabled = true;
     private boolean countDistinctPushdownEnabled = true;
     private boolean grpcEnabled = true;
+    private Optional<URI> proxyRestUri = Optional.empty();
```

Comment:
Rename to proxyUri

Reply:
removed proxyUri to a boolean proxyEnabled to indicate using proxyUri for broker requests as well.